www/static/js/docs.js

```diff
@@ -40,7 +40,7 @@ $(document).ready(function () {
 
     // Table of Contents
     $('#page-toc').toc({
-        'selectors':         'h1,h2,h3,h4,h5,h6', // elements to use as headings
+        'selectors':         'h1,h2', // elements to use as headings
```

Comment:
@dblotsky Its a tradeoff between the semantic meaning and what you want to show in ToC. And, this seems reasonable